src/settings-ui/Settings.UI/PowerToys.Settings.csproj

```diff
@@ -86,10 +86,22 @@
     <Page Update="OOBE\Views\OobeHosts.xaml">
       <Generator>MSBuild:Compile</Generator>
     </Page>
+    <Page Update="OOBE\Views\OobeOverviewAlternate.xaml">
+      <XamlRuntime>$(DefaultXamlRuntime)</XamlRuntime>
+    </Page>
   </ItemGroup>
   <ItemGroup>
     <Page Update="OOBE\Views\OobePowerOCR.xaml">
       <XamlRuntime>$(DefaultXamlRuntime)</XamlRuntime>
     </Page>
   </ItemGroup>
+  <PropertyGroup>
+	<ExperimentationLive>False</ExperimentationLive>
```

Comment:
nit: please make sure to use spaces, as is the prevailing format for this file

Reply:
Do you mean carriage returns between each property/item group?

Reply:
Ah, I mean indenting with spaces instead of the Tab character. :)